[gameplay]/speedometer/client.lua

```diff
@@ -62,6 +62,7 @@ addEventHandler("onClientElementDestroy", root, destroyHandler)
 -- If player vehicle changes in abnormal way (e.g drives into a vehicle pick-up)
 local function onVehicleTypeChange(oldModel, newModel)
     if (getElementType(source) ~= "vehicle") then return end
+    if (getPedOccupiedVehicle(localPlayer) ~= source) then return end
```

Comment:
```suggestion
    if (getPedOccupiedVehicle(localPlayer) ~= source) then return end
```
I don't think `getElementType` check is necessary anymore